.github/workflows/test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       ##########################################################################
       - name: Admin | Build `test` image
         run: |
-          docker build --target test -t "gradido/admin:test" admin/
+          docker build --target test -t "gradido/admin:test" admin/ --build-arg NODE_ENV="test"
```

Comment:
@ulfgebhardt Therefore I changed ENV to ARG in the admin Dockerfile

Reply:
This might have side effects for the application. Do we have checks in the `config/index.ts` - do they still work?